Model Analyzer is crashing

This happens very often the last few weeks. here is an example snipped from the model analyzer logs.

```
Caused by: java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
        at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
        at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
        at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:248)
        at java.base/java.util.Objects.checkIndex(Objects.java:372)
        at java.base/java.util.ArrayList.get(ArrayList.java:458)
        at supply_chain_puzzle___delmia_version20211021.Main.getActionMask(Main.java:4273)
        at supply_chain_puzzle___delmia_version20211021.Main$1ActionMasks.<init>(Main.java:3006)
        ... 9 more
java.io.FileNotFoundException: File '' does not exist

```

We still get the errors even after the model analyzer refactoring, and when we started copying the model analyzer libraries from the [env]-static-training-files buckets.

--- a/nativerl-analyzer/scripts/check_model.sh
+++ b/nativerl-analyzer/scripts/check_model.sh
@@ -22,8 +22,8 @@ fi
 
 cd ${workDir}
 
-export CLASSPATH=$(find ${libDir}/pathmind -iname '*.jar' -print0 | sort -z | xargs --null -i printf "{}:")
-export CLASSPATH=$(find $PWD/lib -iname '*.jar' -print0 | sort -z | xargs --null -i printf "{}:"):${CLASSPATH}
+export CLASSPATH=$(find ${libDir}/pathmind -iname '*.jar' -print0 | sort -z -r | xargs --null -i printf "{}:")
+export CLASSPATH=$(find $PWD/lib -iname '*.jar' -print0 | sort -z -r | xargs --null -i printf "{}:"):${CLASSPATH}
 export CLASSPATH=$PWD:$PWD/model.jar:${CLASSPATH}
 export CLASSPATH=/ma-common.jar:${CLASSPATH}
 

